### ISSUE TITLE
Interview fails on IKEA E1524/E1810

### ISSUE THREAD

**Issue by MrManor** [Koenkk/zigbee2mqtt#21587]
### What happened?

A neighbor placed a wifi thingie on channel wifi 1 forcing me to change channel on my zigbee network from channel 11 to 25. I have successfully re-paired all my devices on the new channel but soon noticed that rigt-left action was missing on all IKEA E1524/E1810 remotes. I then tried to perform a leave/join operation, but after leaving the network any E1524/E1810 device is unable to complete the interview upon joining the network again. 

The led flashingon the device  during interview phase concludes with a double blink like usual. 

I have tried both the current docker vesion and the dev version, no difference.
I also had another remote in a drawer which has never been on my network before, no luck with that one either.

### What did you expect to happen?

Successful join and interview completed.

### How to reproduce it (minimal and precise)

I just try to pair the device but pressing the buttom 4 times as described.

### Zigbee2MQTT version

1.35.3 commit: fe0742a

### Adapter firmware version

6.10.3.0 build 297

### Adapter

EZSP v8, SONOFF ZBDongle-P

### Setup

docker container 

### Debug log

[ikeaJoin-log.txt](https://github.com/Koenkk/zigbee2mqtt/files/14409548/ikeaJoin-log.txt)

**Comment by MrManor** [comment]
The mystery deepens: I visited a friend today and tried to connect my IKEA remotes to his network. The both paired and was interviewed without problems 🤯 
He is running a slightly older Zigbee2MQTT version 1.35.1 commit 9eaaaof, but uses another dongle Coordinator EZSP v12 with revision 7.3.2.0 build 212

**Comment by MrManor** [comment]
I have now replaced my coordinator with a Zonoff Dongle-P, Coordinator type zStack3x0, Coordinator revision 20210708. Again 40 other devices paired without problems, including 4 small IKEA on/off switches type E1743. I got a slightly better result with the IKEA puck now, as I was able to pair 8 of my 11 remotes. The last 3 fails interview totally like show above.

However judging from the Home Assistant webprogrammer, some of the successfully paired ones still misses some random actions. Like missing the Left or Right action.
![puk](https://github.com/Koenkk/zigbee2mqtt/assets/4838974/98b8b7cb-c21d-44a4-9250-5da7d4e9604f)
